airflow/www/static/js/dag/details/taskInstance/MappedInstances.tsx

```diff
@@ -68,7 +68,7 @@ const MappedInstances = ({ dagId, runId, taskId, onRowClicked }: Props) => {
   const data = useMemo(
     () =>
       taskInstances.map((mi) => ({
-        ...mi,
+        mapIndex: mi.renderedMapIndex === null ? mi.mapIndex : mi.renderedMapIndex,
```

Comment:
```suggestion
        mapIndex: mi.renderedMapIndex || mi.mapIndex ,
```

Reply:
There's gonna be a few more spots where I'd like the UI to use the renderedMapIndex. But I can do that in a follow up PR

Reply:
I still need to figure out how to use the integer `mapIndex` and not the `renderedMapIndex` when clicking on the "Logs" button from the grid view too

Reply:
I can handle that part if you want me to.